server/src/main/java/org/elasticsearch/action/admin/cluster/state/ClusterStateResponse.java

```diff
@@ -75,11 +79,24 @@ public ByteSizeValue getTotalCompressedSize() {
         return totalCompressedSize;
     }
 
+    /**
+     * Returns whether waiting for a cluster state wait a metadata version equal or higher than the specified
```

Comment:
Returns whether -> Returns whether the request timed out waiting... and remove timed out from the end of the sentence.